smoke_tests.sh

```diff
@@ -16,19 +16,7 @@ smoke_url_ok $(hostname)/ping
 
 if [ "$ENVIRONMENT_NAME" != "ci" ] && [ "$ENVIRONMENT_NAME" != "dev" ]
   then
-    checks=$(curl -v $(hostname)/status | grep check__name)
-    echo $checks
-    good_checks=$(echo $checks | grep -o ✔ | wc -l)
-    bad_checks=$(echo $checks | grep -o ✘ | wc -l)
-    if [ "$bad_checks" -ne 0 ]; then
-        echo "Not all status checks are ok"
-        exit 2
-    elif [ "$good_checks" -eq 0 ]; then
-        echo "There is not even one status check"
-        exit 3
-    else
-        echo "All checks are ok"
-    fi
+    retry ./status_test.sh 2
```

Comment:
Would we want a `sleep` in there?

Reply:
Can be added if needed (if it becomes clear it's still bootstrapping rather than simple non-determinism). https://github.com/elifesciences/builder-base-formula/blob/master/elife/utils/retry can easily take additional arguments and use an exponential backoff.

Reply:
Yep, worth trying a simple retry as a first step though.